ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TopicInfoCache.java

```diff
@@ -96,7 +96,8 @@
   private static final Set<Pattern> IGNORE_INTERNAL_TOPIC_NAMES_PATTERNS = ImmutableSet.of(
       Pattern.compile(TOPIC_PATTERN_PREFIX + ".*-FK-JOIN-SUBSCRIPTION-REGISTRATION-\\d+-topic"),
       Pattern.compile(TOPIC_PATTERN_PREFIX + ".*-FK-JOIN-SUBSCRIPTION-RESPONSE-\\d+-topic"),
-      Pattern.compile(TOPIC_PATTERN_PREFIX + ".*-FK-JOIN-SUBSCRIPTION-STATE-STORE-\\d+-changelog")
+      Pattern.compile(TOPIC_PATTERN_PREFIX + ".*-FK-JOIN-SUBSCRIPTION-STATE-STORE-\\d+-changelog"),
+      Pattern.compile(TOPIC_PATTERN_PREFIX + ".*-FK-JOIN-OUTPUT-STATE-STORE-\\d+-changelog")
```

Comment:
I don't think we should exclude this topic -- it's not a "internal" topic, but the changelog of the FK-join result if we materialize it.

Reply:
Fixed in the commit titled "chore: do not ignore fk join result changelog in qtt"